Makefile

```diff
@@ -9,8 +9,7 @@ OUTPUT_PACKAGE=Carthage.pkg
 CARTHAGE_EXECUTABLE=./.build/release/carthage
 BINARIES_FOLDER=/usr/local/bin
 
-PWD := $(shell pwd)
-SWIFT_BUILD_FLAGS=--skip-update --configuration release -Xswiftc -suppress-warnings
+SWIFT_BUILD_FLAGS=--configuration release -Xswiftc -suppress-warnings
```

Comment:
I would keep --skip-update

Reply:
Question (if anybody knows): what does that do if a previous resolution doesn’t exist on-disk?

Reply:
I'm guessing it would kick of a resolution.

Leaving this here for future reference if needed: https://forums.swift.org/t/about-reproducible-builds/26957/5?u=tmpz